tools/ci_build/op_registration_utils.py

```diff
@@ -94,6 +94,7 @@ def process_registration(
         start_version: int,
         end_version: typing.Optional[int] = None,
         type: typing.Optional[str] = None,
+        type2: typing.Optional[str] = None,
```

Comment:
there's another usage here:
https://github.com/microsoft/onnxruntime/blob/f3b8130d1a79b54fc6032e3e1800b873847f0b15/tools/ci_build/reduce_op_kernels.py#L135-L143

actually is this still needed? looks like QuantizeLinear (from the example comment below) only has a single type in its registration now.

Reply:
The CUDA registrations still have 2 types. AFAIK we don't do a reduced ops build for CUDA given that's typically a desktop/server scenario but I guess in theory someone might want to do a custom build for an edge device like a Jetson.

Reply:
I registered the QuantizeLinear kernel with two types: `class ONNX_OPERATOR_TWO_TYPED_KERNEL_CLASS_NAME(kCudaExecutionProvider, kOnnxDomain, 19, uint8_t, float, QuantizeLinear);`.